scripts/userstyles.yml

```
@@ -919,4 +920,10 @@ userstyles:
     color: red
     current-maintainers: [*isabelroses, *uncenter]
     past-maintainers: [*elkrien]
+  zen-browser:
```

Just had a discussion with @sgoudham and without appending `-docs` to the name to help the website handle the name duplication here

```suggestion
  zen-browser-docs:
```